// public class MultipleCatchBlock5 {
//     public static void main(String[] args) {
        
//         try{

//             int a[] =  new int [5];
//             a[5] = 30/0;
//         }
//         catch(Exception e){System.out.println("PE");}
//         // catch(ArithmeticException e){System.out.println("t1 completed");}
//         // catch(ArrayIndexOutOfBoundsException e){
//         // }   // Errors 
//     }
    
// }
